tests/mentorship_relation/test_api_send_request.py

```diff
@@ -29,9 +36,10 @@ def test_fail_send_request_bad_mentee_id(self):
                                            headers=auth_header, content_type='application/json',
                                            data=json.dumps(test_payload))
         self.assertEqual(404, actual_response.status_code)
+        self.assertEqual(1, len(outbox))
```

Comment:
`outbox` isn't defined here.

Reply:
@gaurivn can you please define how to get the outbox from the user email

Reply:
@isabelcosta @BadduCoder, can I remove the outbox line

Reply:
@gaurivn I did a quick search on google to try to figure out how to use the `outbox` concept, and I found this in the official documentation of Flask-Mail: https://pythonhosted.org/Flask-Mail/#unit-tests-and-suppressing-emails
Can you see if you can fix the `outbox` usage in this test, before deciding to not use this?

This issue goal is about using something in the code to figure out if the email is being sent, and I think this can be done through this `outbox` concept